diff --git a/node_modules/@nuxt/vite-builder/dist/index.mjs b/node_modules/@nuxt/vite-builder/dist/index.mjs
index b45f2e5..f021921 100644
--- a/node_modules/@nuxt/vite-builder/dist/index.mjs
+++ b/node_modules/@nuxt/vite-builder/dist/index.mjs
@@ -630,16 +630,7 @@ function listenAndRestrict(server, socketPath) {
 		server.listen(socketPath, () => {
 			try {
 				fs.chmodSync(socketPath, 384);
-			} catch (error) {
-				bundlerDiagnostics.NUXT_B7018({ cause: error });
-				server.close();
-				try {
-					fs.rmSync(dirname(socketPath), {
-						recursive: true,
-						force: true
-					});
-				} catch {}
-			}
+			} catch {}
 		});
 	} finally {
 		process.umask(previousUmask);
